includes/class-suggestion-engine.php

```
@@ -0,0 +1,347 @@
+			$ucss = (float) ( $diagnostics['unused-css-rules']['score'] ?? 1.0 );
+			if ( $ucss < 0.9 ) {
+				$suggestions[] = self::make_score(
+					'unused_css',
+					$ucss,
+					'open_file_optimization_tab',
+					'Unused CSS'
+				);
+			}
+
+			// --- Text compression ---
+			$tc = (float) ( $diagnostics['uses-text-compression']['score'] ?? 1.0 );
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $tc. Configured minimum length is 3. (undefined)

(ShortVariable)